modules/nodes-descheduler-installing.adoc

```diff
@@ -22,7 +22,7 @@ endif::[]
 . Log in to the {product-title} web console.
 . Create the required namespace for the Kube Descheduler Operator.
 .. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
-.. Enter `openshift-kube-descheduler-operator` in the *Name* field and click *Create*.
+.. Enter `openshift-kube-descheduler-operator` in the *Name* field, enter `openshift.io/cluster-monitoring=true` in the *Labels* field and click *Create*.
```

Comment:
@ingvagabund Would you mind adding a comma after "field", and then squashing back to 1 commit?

Otherwise this LGTM, thanks!